core/src/main/resources/schema/infinispan-config-11.0.xsd

```diff
@@ -1323,8 +1335,59 @@
         </xs:element>
       </xs:choice>
     </xs:sequence>
+    <xs:attribute name="max-size" type="xs:string">
+      <xs:annotation>
+        <xs:documentation>
+           Defines the size of the data container in a byte quantity plus an optional multiplier.
+           Supported multipliers are kB (kilobytes), MB (megabytes), GB (gigabytes) and TB (terabytes).
+           Eviction occurs either the amount of memory exceeds the maximum size.
+        </xs:documentation>
+      </xs:annotation>
+    </xs:attribute>
+    <xs:attribute name="max-count" type="xs:long" default="-1">
+      <xs:annotation>
+        <xs:documentation>
+          Defines the max number of entries in the data container, before eviction occurs.
+        </xs:documentation>
+      </xs:annotation>
+    </xs:attribute>
+    <xs:attribute name="eviction-strategy" type="tns:eviction-strategy">
+      <xs:annotation>
+        <xs:documentation>
+          Specifies a strategy for evicting cache entries. Eviction always
+          takes place when you define either the max-size or the max-count
+          (but not both) for the data container.
```

Comment:
Should probably mention that REMOVE is used if no strategy is defined but size or count are.